src/glue/mod.rs

```diff
@@ -49,10 +52,66 @@ impl Glue {
                         })
                         .collect::<Vec<Vec<String>>>(),
                 )
-                .collect()),
-            _ => Err(ExecuteError::QueryNotSupported.into()),
+                .collect())
+        } else {
+            Err(ExecuteError::QueryNotSupported.into())
         }
     }
+
+    pub fn insert_vec(
+        &mut self,
+        table_name: String,
+        columns: Vec<String>,
+        rows: Vec<Vec<Value>>,
+    ) -> Result<Payload> {
+        // TODO: Make this more efficient and nicer by checking the way we execute
+        let table_name = ObjectName(vec![Ident {
+            value: table_name,
+            quote_style: None,
+        }]);
+        let columns = columns
+            .into_iter()
+            .map(|name| Ident {
+                value: name,
+                quote_style: None,
+            })
+            .collect();
+        let sqlparser_rows: Vec<Vec<Expr>> = rows
+            .into_iter()
+            .map(|row| {
+                row.into_iter()
+                    .map(|cell| {
+                        Expr::Value(match cell {
+                            Value::Null => Literal::Null,
+                            Value::Bool(value) => Literal::Boolean(value),
+                            Value::I64(value) => Literal::Number(value.to_string(), false),
+                            Value::F64(value) => Literal::Number(value.to_string(), false),
+                            Value::Str(value) => Literal::SingleQuotedString(value),
+                        })
+                    })
+                    .collect()
+            })
+            .collect();
```

Comment:
Is this an intended behavior that you decide to get `Vec<Vec<Value>>` rather than `Vec<Vec<sqlparser::ast::Value>>`?
If it is because of specific use cases, then I'll accept.

Reply:
Yup, my thinking is that we don't want GlueSQL users to have to play around with `sqlparser`, we want them to interface with `Value`.

Reply:
Ok :)